lib/bundler/source/rubygems.rb

```diff
@@ -158,7 +158,9 @@ def install(spec, opts = {})
           installed_spec.loaded_from = loaded_from(spec)
         end
         spec.loaded_from = loaded_from(spec)
-        ["Installing #{version_message(spec)}", spec.post_install_message]
+        install_message = "Installing #{version_message(spec)}"
+        install_message << " with native extensions" unless spec.extensions.empty?
```

Comment:
Comma before "with native extensions"? Eh, I'm fine with how it was before, which this replicates exactly.